src/app/qgsprojectproperties.cpp

```diff
@@ -2475,6 +2497,74 @@ void QgsProjectProperties::mButtonAddColor_clicked()
   mTreeProjectColors->addColor( newColor, QgsSymbolLayerUtils::colorToName( newColor ) );
 }
 
+void QgsProjectProperties::setEndAsStartButton_clicked()
```

Comment:
It may be useful when we want to show temporal status of different single date time instant layers using the instant set from the project time settings.